projects/driver_controls/inc/ads1015.h

```diff
@@ -38,6 +46,10 @@ typedef struct Ads1015Storage {
   uint8_t pending_channel_bitset;
   Ads1015Callback channel_callback[NUM_ADS1015_CHANNELS];
   void *callback_context[NUM_ADS1015_CHANNELS];
+
```

Comment:
Nit: No newlines in struct definitions unless comments? Unless we want to change our style guide in which I would be ok with:

Reply:
Do we have something in our style guide about not having newlines in struct definitions? I find that it helps for large structs.